### ISSUE TITLE
When sprig, closing HTML tag in <script> get removed

### ISSUE THREAD

**Issue by hongc-cc** [putyourlightson/craft-sprig#34]
### Describe the bug
When `sprig`, closing HTML tag in `<script>` get removed

### To reproduce
Live test case: https://asiaspa.com/form?123

Simplified test case
````
{# text.twig #}

{% include "_script.twig" %}
{{ sprig ("_script.twig") }}
````

````
{# _script.twig #}

<span>placeholder</span>
<script type="text/javascript"><span>placeholder</span></script>
````

Rendered Result:
````
<!-- test.twig -->
<span>placeholder</span>
<script type="text/template"><p>placeholder</p></script>

<span>placeholder</span>
<script type="text/template"><p>placeholder</script>
````

### Versions
- Plugin version:1.0.0-beta.16
- Craft version:  3.5.11.1

**Comment by bencroker** [comment]
What are you trying to do with this code? JavaScript code is generally placed inside of `<script>` tags, not HTML.

```html
<script type="text/javascript"><span>placeholder</span></script>
```

Either way, you should try to avoid having `<script>` tags inside of Sprig components and keep them in the main template only.

**Comment by hongc-cc** [comment]
Agreed. Yet is the stripping behavior normal / expected?
I think putting HTML snippets in `<script type="text/javascript">` is a common approach 
when developers dont want to put too much HTML in javascript file.

Issue originated from another plugin:
https://github.com/verbb/formie/issues/131

**Comment by bencroker** [comment]
The markup is being modified by the DOM library that Sprig uses to walk the DOM tree looking for attributes. I'll look into what can be done about this.

**Comment by bencroker** [comment]
Your code appears to be invalid HTML so Sprig is not really at fault here. I suggest you use the [template tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template), which is valid HTML and works fine with Sprig.

```html
<template><p>placeholder</p></template>
```

**Comment by hongc-cc** [comment]
i think `<script type="text/template"><span>placeholder</span></script>` is not invalid HTML?

**Comment by engram-design** [comment]
Just weighing in here. It's not an option to use `<template>` for Formie as part of our commitment to front-end support. It's not supported in IE10/11 for example.

I suppose as @bencroker mentions, its being modified by the DOM library, so it's probably an issue on their end.

Formie's repeater field uses `<script type="text/template">some-html</script>` in its templates.

**Comment by bencroker** [comment]
Ok reopening and I’ll see what can be done to make this work.

**Comment by bencroker** [comment]
Fixed in https://github.com/putyourlightson/craft-sprig/commit/bf4a9ab7d073661d6d3b71601071a6bf02a72e64 and released in version 1.0.0-beta.17.
